### ISSUE TITLE
[Follow person]

### ISSUE THREAD

**Issue by OscarMrZ** [JdeRobot/RoboticsAcademy#2415]
Currently, the teleoperation of the robot in the Follow Person exercise is done through a separate tool that starts a dedicated websocket server. This isn't currently operative and introduces unnecesary complexities in the RAM. I think this should be handled from the exercise template.

**Comment by jmplaza** [comment]
Yes, the teleoperation will performed with the GUI.py receiving the pressed keys from the browser through the regular websocket connecting the RAM (and the gui-broker) and the browser. This way no extra port will be required, no extra component will be required either and the functionality will be solved in the python template, not in the manager.
